src/main.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+main() {
+    sudo xbps-install $(find ./src/programs -type f | sed 's!.*/!!' | tr '\n' ' ')
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Quote this to prevent word splitting. [SC2046](https://github.com/koalaman/shellcheck/wiki/SC2046)